# 城市总数、连接这些城市的道路数
int32 number_of_cities
int32 number_of_roads
int32 therehold
RoadInfo[] infos
int32 src_city
int32 des_city
